user_service/user_service/views/views.py

```diff
@@ -0,0 +1,273 @@
+    refresh_jwt = jwt.encode(
+        {
+            "exp": datetime.datetime.utcnow() + datetime.timedelta(hours=3),
+            "user_id": str(user_id),
+            "created_at": str(datetime.datetime.utcnow()),
+        },
+        SECRET,
+    ).decode("utf-8")
+
+    return access_jwt, refresh_jwt
+
+
+def jwt_decode(jwt_token):
+    """Function to check fact of jwt expiration.
+    Returns decoded payload of jwt if it's OK or string 'Expired jwt'"""
+
+    try:
+        decoded_jwt = jwt.decode(jwt_token, SECRET, algorithms="HS256")
+    except jwt.ExpiredSignatureError:
+        decoded_jwt = "Expired jwt"
+
+    return decoded_jwt
+
+
+def login_required(func):
+    """Decorator-function allows to pass to views for authorised users only"""
+
+    async def wrapped(self, *args, **kwargs):
+
+        # get access token string from header
+        access_check = self.headers.get("Authorization", None)
+        if access_check is None:
+            return aiohttp.web.Response(
+                text=json.dumps("User unauthorised"), status=403
+            )
+
+        # remove 'Bearer ' string from access token string
+        access_check = access_check[7:]
+
+        # check access token in session
+        session = await get_session(self)
+        try:
+            session[access_check]
+        except KeyError:
+            return aiohttp.web.Response(
+                text=json.dumps("User unauthorised (wrong token)"), status=403
+            )
+
+        # check expiration of access token and decode access token payload
+        decoded_access = jwt_decode(access_check)
+
+        if decoded_access == "Expired jwt":
+            # that 'if' for tokens refresh feature
+            if self.path == "/refresh":
+                return await func(self, access_check, *args, **kwargs)
+            else:
+                return aiohttp.web.Response(
+                    text=json.dumps("You need to re-login"), status=403
+                )
+
+        return await func(self, access_check, *args, **kwargs)
+
+    return wrapped
+
+
+async def register(request):
+    """View for user registration. Obligatory params: username, password.
+    User Id is creating on model initialisation.
+    Password hashes with argon2 also on initialisation"""
+
+    # getting user info from request
+    try:
+        username = request.query["username"]
+        password = request.query["password"]
+    except KeyError:
+        return aiohttp.web.Response(
+            text=json.dumps("Username and password required"), status=403
+        )
+
+    # user object creation
+    user = model.User(username, argon2.PasswordHasher().hash(password))
+
+    # saving user to DB with check of username duplication
+    async with request.app["db"].acquire() as conn:
+        try:
+            await conn.execute(
+                sa.insert(model.User).values(
+                    id=user.id, username=user.username, password=user.password
+                )
+            )
```

Comment:
У тебя у id есть default. Поэтому не имеет смысла заранее создавать пользователя. Вообще логика довольно странная. 

```suggestion
    # saving user to DB with check of username duplication
    async with request.app["db"].acquire() as conn:
        try:
            await conn.execute(
                sa.insert(model.User).values(
                  username=username, 
                  password= argon2.PasswordHasher().hash(password)
                )
            )
```